.github/actions/baas-test-server/action.yml

```diff
@@ -0,0 +1,20 @@
+name: Start BaaS test server
```

Comment:
Note that this action is coupled to the Realm JS repo, since it depends on the `baas-test-server` CLI and its dependencies being installed ahead of being invoked.